internal/reconcilers/artifacts.go

```diff
@@ -154,93 +150,19 @@ func (e *Reconciler) handleArtifactsReconcilerEvent(ctx context.Context, evt *Re
 		return err
 	}
 
-	// create artifact verifier
-	artifactVerifier, err := verifier.NewVerifier(
-		verifier.VerifierSigstore,
-		sigstore.SigstorePublicTrustedRootRepo,
-		container.WithAccessToken(cli.GetToken()), container.WithGitHubClient(cli))
-	if err != nil {
-		return fmt.Errorf("error getting sigstore verifier: %w", err)
-	}
-	defer artifactVerifier.ClearCache()
-
 	for _, artifact := range artifacts {
 		// store information if we do not have it
+		typeLower := strings.ToLower(artifact.GetPackageType())
 		newArtifact, err := e.store.UpsertArtifact(ctx,
 			db.UpsertArtifactParams{RepositoryID: repository.ID, ArtifactName: artifact.GetName(),
-				ArtifactType: artifact.GetPackageType(), ArtifactVisibility: artifact.GetVisibility()})
+				ArtifactType: typeLower, ArtifactVisibility: artifact.GetVisibility()})
 
 		if err != nil {
 			// just log error and continue
 			log.Printf("error storing artifact: %v", err)
 			continue
 		}
 
-		// remove older versions
```

Comment:
Are we simply keeping all the artifact versions for now?

Reply:
No, rather we are not keeping anything at all now.

Reply:
(this might have caused the confusion above about Artifacts and ArtifactVersions that were stored in two different tables)